sp/src/game/client/mapbase/c_trigger_portal.cpp

```diff
@@ -0,0 +1,31 @@
+//========= Copyright Valve Corporation, All rights reserved. ============//
```

Comment:
As part of the v5.0 update's new contribution guidelines, this should now use the Mapbase header.